pkg/webservice/handlers.go

```diff
@@ -869,3 +870,53 @@ func getGroupResourceUsage(w http.ResponseWriter, r *http.Request) {
 		buildJSONErrorResponse(w, err.Error(), http.StatusInternalServerError)
 	}
 }
+
+func getEvents(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+	eventSystem := events.GetEventSystem()
+	if eventSystem == nil {
+		buildJSONErrorResponse(w, "Event system is disabled", http.StatusBadRequest)
+		return
+	}
+
+	count := uint64(10000)
+	var fromId uint64
+	vars := httprouter.ParamsFromContext(r.Context())
+	if vars != nil {
+		if countStr := vars.ByName("count"); countStr != "" {
+			c, err := strconv.ParseInt(countStr, 10, 64)
+			if err != nil {
+				buildJSONErrorResponse(w, err.Error(), http.StatusBadRequest)
+				return
+			}
+			if c <= 0 {
+				buildJSONErrorResponse(w, fmt.Sprintf("Illegal number of events: %d", c), http.StatusBadRequest)
+				return
+			}
+			count = uint64(c)
+		}
+
+		if fromIdStr := vars.ByName("fromId"); fromIdStr != "" {
```

Comment:
I think we should name this parameter "start" instead of fromID. It's simpler to type and no casing issues to address. As such, startStr should be the variable name used.